batch/batches/LiveToVod/KAsyncLiveToVod.class.php

```diff
@@ -37,7 +37,25 @@ protected function getJobType()
 	 */
 	protected function exec(KalturaBatchJob $job)
 	{
-        return $this->copyCuePoint($job, $job->data);
+		$jobRes = $this->copyCuePoint($job, $job->data);
+        try {
+            $liveEntryId = $job->data->liveEntryId;
+            $liveEntry = KBatchBase::$kClient->baseEntry->get($liveEntryId);
+            /** @var KalturaLiveStreamEntry $liveEntry */
+            $recordStatus = $liveEntry->recordingStatus;
+            $shouldAutoArchive = $liveEntry->recordingOptions->shouldAutoArchive;
+            if ($recordStatus == KalturaRecordStatus::PER_SESSION && $shouldAutoArchive == KalturaNullableBoolean::TRUE_VALUE) {
+                $archiveOptions = new KalturaLiveEntryArchiveOptions();
+                $archiveOptions->notDeletedCuePointTags = "entry-reset-mode,webcast:internal-state,player-qna-settings-update,select-deck-document,poll-data,timeline-assets-state,rtc-settings";
```

Comment:
Better move the tags to the recording options and make thus making it more generic.